### PR TITLE
Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/arrays.xml [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+        <item>总是(及视频)</item>
     </string-array>
     <string-array name="auto_rotation_entries">
         <item>自动</item>
-        <item>默认横屏</item>
-        <item>默认竖屏</item>
-        <item>Device</item> <!-- TODO -->
+        <item>默认横向</item>
+        <item>默认纵向</item>
+        <item>设备</item> <!-- TODO -->
```
Comment (sfan5) [inline app/src/main/res/values-zh-rCN/arrays.xml:12]: `<!-- TODO -->` was supposed to indicate that this translation is incomplete. You can remove it.

File: app/src/main/res/values-zh-rCN/arrays.xml [hunk 2]
```diff
@@ -18,10 +18,22 @@
...
-    </string-array>
-</resources>
+        <item>stats.lua：缓存</item> <!-- TODO -->
+
+    <string-array name="gestures_entries">
+        <item>无</item>
+        <item>进度</item>
+        <item>音量</item>
+        <item>亮度</item>
+    </string-array>    </string-array>
```
Comment (sfan5) [inline app/src/main/res/values-zh-rCN/arrays.xml:32]: duplicate XML tag

File: app/src/main/res/values-zh-rCN/arrays.xml [hunk 3]
```diff
@@ -18,10 +18,23 @@
...
         <item>stats.lua：常规</item>
         <item>stats.lua：计时</item>
-        <item>stats.lua：Cache</item> <!-- TODO -->
+        <item>stats.lua：缓存</item>
     </string-array>
-</resources>
+
+    <string-array name="gestures_entries">
+        <item>无</item>
+        <item>进度</item>
```
Comment (sfan5) [inline app/src/main/res/values-zh-rCN/arrays.xml:30]: VLC translates "seek" as 定位
No idea if that's better as I don't speak Chinese, just thought you might want to consider this suggestion.

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -10,25 +10,52 @@
     <string name="dialog_save">保存</string>
     <string name="dialog_yes">是</string>
     <string name="dialog_no">否</string>
+    <string name="dialog_prev">上一个媒体</string>
+    <string name="dialog_next">下一个媒体</string>
```
Comment (sfan5) [inline app/src/main/res/values-zh-rCN/strings.xml:14]: These two are supposed to be general terms, no relation to media.

**Comment by sfan5** [comment]
If you're done making changes please also squash the commits into a single one, then I can merge.

File: app/src/main/res/values-zh-rCN/arrays.xml (hunk 3 above)
Comment (linsui) [reply app/src/main/res/values-zh-rCN/arrays.xml]: Thanks!

**Pull request merged**